travis/module.yml

```diff
@@ -69,3 +69,4 @@ script:
   - cd $HOME/test-root && composer server start
   # Run Codeception for module tests, if test suites are found.
   - cd $HOME/test-root && composer dev-tools codecept run -p vendor/$ALTIS_PACKAGE/tests
+  - cd $HOME/test-root && composer dev-tools lintdocs -l vendor/$ALTIS_PACKAGE all
```

Comment:
I think this will fail the build in Travis if the lintdocs command returns a non-zero code. Do we want to add some `|| true` here while we know they're mostly failing?

Reply:
Actually at the moment the code is hard-coded to always return a zero 
https://github.com/humanmade/altis-dev-tools-command/pull/45/files#diff-82d56c2b43806fb465261bae191ce8148869b9023625b16e2d0c44bd37dfa1cfR1155

Reply:
Ah okay that's fine then!